README.md

```diff
@@ -1,2 +1,53 @@
-# esxi-exporter
-Prometheus exporter, which utilises the VMware SDK to get metrics from VMware ESXi.
+# ESXi-exporter
+Prometheus exporter, which utilises the _VMware SDK_ and _SSH_ to get metrics from _VMware ESXi-HostSystems_.
+
+## About
+This exporter contains a `critical service collector` and a `overall state collector`. It monitors the service state of services like `hostd` or `ntpd` and gathers the _ESXi-HostSystem_ `overallState` from _vCenters_. 
+
+
+## Getting started
+
+1. Configure the project
+    - Credentials are passed by environment variables.
+    - `config.yaml` contains some static configuration
+    - There are command-line options for more console output
+2. Run `exporter.py` with _python3_
+
+**Environment variables**
+- `VCENTER_USER` the vCenter username
+- `VCENTER_PASSWORD` the vCenter password
+- `VCENTER_MPW` if set, master_password mechanism will be used.
+- `ESXI_USER` the ESXi-host ssh username
+- `ESXI_PASSWORD` the ESXi-host ssh password
+- `ATLAS_FILE` the path to the atlas file containing esxi_hosts and vcenters

```

Comment:
Can you please link to the atlas project? https://github.com/sapcc/atlas

Reply:
done.